test/Table_Tests/src/Common_Table_Operations/Column_Operations_Spec.enso

```diff
@@ -224,7 +225,67 @@ spec setup =
             s1.ends_with "a" . to_vector . should_equal [False, False, False, False, Nothing]
             s1.ends_with "b" Case_Sensitivity.Insensitive . to_vector . should_equal [False, False, False, True, Nothing]
 
-    Test.group prefix+"Colum Operations - Names" <|
+    Test.group prefix+"Column Operations - Text Replace" <|
+        if setup.is_database.not then
```

Comment:
I know `replace` is not yet supported by DB, but I think we should add least add a `replace` function always throwing `Unsupported...` to keep API compatibility. If I'm not mistaken it's not there yet, could it be added?

Reply:
yes am going to add it.